inventory/idm-server/group_vars/idm-server.yml

```diff
@@ -12,6 +12,6 @@ rhsm_repos:
 
 idm_master_hostname: idm.example.com
 idm_domain: example.com
-idm_realm: example.com
+idm_realm: EXAMPLE.COM
```

Comment:
@bvkin This is good to change it in the example, but I think it would also be good to have somewhere where we set `idm_realm = "{{ idm_realm|upper }}"`